pysb/simulator/base.py

```diff
@@ -224,27 +234,72 @@ class SimulationResult(object):
         A set of species trajectories from a simulation. Should either be a
         list of 2D numpy arrays or a single 3D numpy array.
 
-    Attributes
```

Comment:
`@property` method docstrings get listed with methods (see [base_filename from the BNG interface for an example](http://pysb.readthedocs.io/en/latest/modules/bng.html#pysb.bng.BngBaseInterface.base_filename)). Having an Attributes section means they get listed twice and documentation is duplicated. Alternatively, they could all be listed under Attributes, and have a method docstring that says "See Attributes for description". Listing docstrings under each method seems cleaner to me though, and nicer when reading the code directly.